<launch>
	<group ns="vision">
		<node pkg="extrinsic_calibrator" type="extrinsiccalibrator.py" name="extrinsic_calibrator">
			<param name="board_cols" value="8"/>
			<param name="board_rows" value="6"/>
			<param name="board_size" value="0.062"/>

			<remap from="camera0" to="left"/>
			<remap from="camera1" to="middle"/>
		</node>
	</group>
</launch>
